Test failing

Compiling version `1.6.1`:

```
ok      github.com/neilberkman/clippy   0.011s
--- FAIL: TestFileMode (0.55s)
    --- FAIL: TestFileMode/text_file_default (0.37s)
        main_test.go:49: clippy failed: exit status 1
            Output: Error: Could not copy file ../../test-files/sample.txt: could not copy file >
    --- FAIL: TestFileMode/text_file_with_-t_flag (0.06s)
        main_test.go:49: clippy failed: exit status 1
            Output: Error: Could not copy file ../../test-files/sample.txt: could not copy text >
    --- FAIL: TestFileMode/elixir_file_default (0.03s)
        main_test.go:49: clippy failed: exit status 1
            Output: Error: Could not copy file ../../test-files/code.exs: could not copy file to>
    --- FAIL: TestFileMode/elixir_file_with_-t_flag (0.03s)
        main_test.go:49: clippy failed: exit status 1
            Output: Error: Could not copy file ../../test-files/code.exs: could not copy text to>
    --- FAIL: TestFileMode/pdf_file (0.03s)
        main_test.go:49: clippy failed: exit status 1
            Output: Error: Could not copy file ../../test-files/test.pdf: could not copy file to>
    --- FAIL: TestFileMode/png_file (0.03s)
        main_test.go:49: clippy failed: exit status 1
            Output: Error: Could not copy file ../../test-files/minimal.png: could not copy file>
--- FAIL: TestStreamMode (0.08s)
    --- FAIL: TestStreamMode/text_stream (0.03s)
        main_test.go:96: clippy failed: exit status 1
            Output: Error: Could not copy from stdin: could not copy text to clipboard: failed t>
    --- FAIL: TestStreamMode/text_file_piped (0.03s)
        main_test.go:96: clippy failed: exit status 1
            Output: Error: Could not copy from stdin: could not copy text to clipboard: failed t>
    --- FAIL: TestStreamMode/binary_file_piped (0.03s)
        main_test.go:96: clippy failed: exit status 1
            Output: Error: Could not copy from stdin: could not copy file to clipboard: failed t>
--- FAIL: TestMultipleFiles (0.03s)
    --- FAIL: TestMultipleFiles/copy_multiple_files (0.03s)
        main_test.go:112: clippy failed: exit status 1
            Output: Error: Could not copy files: could not copy files to clipboard: failed to wr>
...skipping...
    --- FAIL: TestMultipleFiles/copy_multiple_files (0.03s)
        main_test.go:112: clippy failed: exit status 1
            Output: Error: Could not copy files: could not copy files to clipboard: failed to wr>
--- FAIL: TestFlags (0.03s)
    --- FAIL: TestFlags/silent_by_default (0.00s)
        main_test.go:143: Failed to write test config: open /homeless-shelter/.clippy.conf: no s>
    --- FAIL: TestFlags/verbose_flag (0.03s)
        main_test.go:169: clippy failed: exit status 1
--- FAIL: TestPipelines (2.57s)
    --- FAIL: TestPipelines/echo_text_to_clippy (0.04s)
        main_test.go:253: Unexpected error: exit status 1
            Output: Error: Could not copy from stdin: could not copy text to clipboard: failed t>
        main_test.go:257: Expected output to contain "Copied content from stream using smart det>
    --- FAIL: TestPipelines/cat_text_file_to_clippy (0.03s)
        main_test.go:253: Unexpected error: exit status 1
            Output: Error: Could not copy from stdin: could not copy text to clipboard: failed t>
        main_test.go:257: Expected output to contain "Copied content from stream using smart det>
    --- FAIL: TestPipelines/cat_binary_file_to_clippy (0.03s)
        main_test.go:253: Unexpected error: exit status 1
            Output: Error: Could not copy from stdin: could not copy file to clipboard: failed t>
        main_test.go:257: Expected output to contain "Copied content from stream using smart det>
    --- FAIL: TestPipelines/find_and_xargs_single_file (0.08s)
        main_test.go:253: Unexpected error: exit status 123
            Output: Error: Could not copy file ../../test-files/sample.txt: could not copy file >
        main_test.go:257: Expected output to contain "Copied file reference", got: Error: Could >
    --- FAIL: TestPipelines/find_and_xargs_multiple_files (0.08s)
        main_test.go:253: Unexpected error: exit status 123
            Output: Error: Could not copy files: could not copy files to clipboard: failed to wr>
        main_test.go:257: Expected output to contain "Copied", got: Error: Could not copy files:>
    --- FAIL: TestPipelines/curl_to_clippy_(simulated) (0.03s)
        main_test.go:253: Unexpected error: exit status 1
            Output: Error: Could not copy from stdin: could not copy file to clipboard: failed t>
        main_test.go:257: Expected output to contain "Copied content from stream using smart det>
    --- FAIL: TestPipelines/empty_input (2.07s)
        main_test.go:253: Unexpected error: exit status 1
            Output: Error: Failed to clear clipboard: clipboard operation timed out
        main_test.go:257: Expected output to contain "Clipboard cleared", got: Error: Failed to >
    --- FAIL: TestPipelines/multiline_text (0.07s)
        main_test.go:253: Unexpected error: exit status 1
            Output: Error: Could not copy from stdin: could not copy text to clipboard: failed t>
        main_test.go:257: Expected output to contain "Copied content from stream using smart det>
    --- FAIL: TestPipelines/command_output_pipe (0.05s)
        main_test.go:253: Unexpected error: exit status 1
            Output: Error: Could not copy from stdin: could not copy text to clipboard: failed t>
        main_test.go:257: Expected output to contain "Copied content from stream using smart det>
    --- FAIL: TestPipelines/base64_encoded_binary (0.04s)
        main_test.go:253: Unexpected error: exit status 1
            Output: Error: Could not copy from stdin: could not copy text to clipboard: failed t>
        main_test.go:257: Expected output to contain "Copied content from stream using smart det>
    --- FAIL: TestPipelines/json_data (0.04s)
        main_test.go:253: Unexpected error: exit status 1
            Output: Error: Could not copy from stdin: could not copy text to clipboard: failed t>
        main_test.go:257: Expected output to contain "Copied content from stream using smart det>
--- FAIL: TestConfigFile (0.00s)
    main_test.go:282: Failed to write test config: open /homeless-shelter/.clippy.conf: no such >
FAIL
FAIL    github.com/neilberkman/clippy/cmd/clippy        7.213s
FAIL
```

--- a/cmd/clippy/main.go
+++ b/cmd/clippy/main.go
@@ -7,6 +7,7 @@ import (
 	"io"
 	"os"
 	"path/filepath"
+	"runtime"
 	"strings"
 	"time"
 
@@ -38,6 +39,13 @@ var (
 )
 
 func main() {
+	// Clippy only works on macOS
+	if runtime.GOOS != "darwin" {
+		fmt.Fprintf(os.Stderr, "Error: Clippy only works on macOS (detected: %s)\n", runtime.GOOS)
+		fmt.Fprintln(os.Stderr, "Clippy uses macOS-specific clipboard APIs and frameworks.")
+		os.Exit(1)
+	}
+
 	// Preprocess args to convert "-r 3" to "-r=3" for Cobra compatibility
 	os.Args = preprocessArgs(os.Args)
 

--- a/cmd/clippy/main_test.go
+++ b/cmd/clippy/main_test.go
@@ -5,11 +5,17 @@ import (
 	"os"
 	"os/exec"
 	"path/filepath"
+	"runtime"
 	"strings"
 	"testing"
 )
 
 func TestMain(m *testing.M) {
+	// Clippy only works on macOS
+	if runtime.GOOS != "darwin" {
+		panic("Clippy tests require macOS (detected: " + runtime.GOOS + ")")
+	}
+
 	// Build the binary for testing
 	cmd := exec.Command("go", "build", "-o", "clippy_test", ".")
 	if err := cmd.Run(); err != nil {

--- a/cmd/pasty/main.go
+++ b/cmd/pasty/main.go
@@ -6,6 +6,7 @@ import (
 	"fmt"
 	"os"
 	"path/filepath"
+	"runtime"
 
 	"github.com/neilberkman/clippy"
 	"github.com/neilberkman/clippy/cmd/internal/common"
@@ -24,6 +25,13 @@ var (
 )
 
 func main() {
+	// Pasty only works on macOS
+	if runtime.GOOS != "darwin" {
+		fmt.Fprintf(os.Stderr, "Error: Pasty only works on macOS (detected: %s)\n", runtime.GOOS)
+		fmt.Fprintln(os.Stderr, "Pasty uses macOS-specific clipboard APIs and frameworks.")
+		os.Exit(1)
+	}
+
 	var rootCmd = &cobra.Command{
 		Use:   "pasty [destination]",
 		Short: "Smart paste tool for macOS",

